fastlane/Helpers/Match.swift

```diff
@@ -27,6 +27,7 @@ enum Match {
                 type: type.match,
                 readonly: .userDefined(!isForce),
                 appIdentifier: [environment.bundleId],
+                apiKey: .userDefined(Constant.apiKey),
```

Comment:
I think we do not recommend to use apiKey for match like that. Or in case we must do it, please check `isForce` too

Reply:
I added checking `isForce` at https://github.com/nimblehq/ios-templates/pull/540/commits/a3d7e1fdcc8cefd41513e9cb1094bdc8bf1647b0 🙏